pnp/cli-microsoft365
# Table list command. Solves #3653

- Table list command. Solves #3653

## How to test

Create a new [Azure AD App registration](https://pnp.github.io/cli-microsoft365/user-guide/using-own-identity/) that you can use with the CLI for Microsoft. Provide this app the at least: 
- Azure Service Management - `user_impersonation` (to access Power Apps API) 
- Dynamics CRM - `user_impersonation` (to access Dynamics) 
- SharePoint - `AllSites.FullControl` to read tenant admin site

Use this new App Registration to sign in to the CLI `m365 login --appId yourAppId --tenant yourTenantId`

Then you should be able to run this command :)

## ⚠️ Alert 
This can only go live with support from @waldekmastykarz  as he can change our App Registration to include the Dynamics CRM - `user_impersonation` permission that is currently not yet available in the CLI App Registration
